v2.1/import-data.md

```diff
@@ -77,10 +77,94 @@ For reference, CockroachDB uses these defaults:
 - `[port]`: **26257**
 - `[user]`: **root**
 
+
+## Import from MySQL Dump
+
+{% include experimental-warning.md %}
+
+<span class="version-tag">New in v2.1:</span> This section has instructions for getting data from MySQL dump files into CockroachDB using [`IMPORT`](import.html). It uses the [employees data set](https://github.com/datacharmer/test_db) that is also used in the [MySQL docs](https://dev.mysql.com/doc/employee/en/).
+
+- [Step 1. Dump the MySQL database](#step-1-dump-the-mysql-database)
+- [Step 2. `IMPORT` the dump files](#step-2-import-the-dump-files)
+
+### Step 1. Dump the MySQL database
+
+Issue a command like the following (modified as needed for your environment):
+
+{% include copy-clipboard.html %}
+~~~ sh
+$ mysqldump -uroot --tab=/tmp employees --fields-enclosed-by='"' --fields-escaped-by='\'
+~~~
+
+Always specify an escape character when running `mysqldump --tab`. If your data happens to include any of the special characters, the dumped data cannot be decoded correctly if they are not escaped. We also recommend specifying an enclose character.
+
+Your MySQL user will need permission to execute [`SELECT INTO OUTFILE`](https://dev.mysql.com/doc/refman/8.0/en/select.html), as required by [`mysqldump --tab`](https://dev.mysql.com/doc/refman/8.0/en/mysqldump-delimited-text.html).
+
+If you get the following error, you will need to change the value of MySQL's [`secure_file_priv` system variable](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_secure_file_priv):
+
+~~~
+mysqldump: Got error: 1290: The MySQL server is running with the --secure-file-priv option so it cannot execute this statement when executing 'SELECT INTO OUTFILE'
+~~~
+
+### Step 2. `IMPORT` the dump files
+
+To import the MySQL dump file for a table, issue an `IMPORT` statement like the one shown below. You must use the same settings for field enclosure and escape characters that you provided to `mysqldump --tab` in the previous step.
+
+You will need to look at the dump's `foo.sql` file that corresponds to the `foo.txt` data file and translate the MySQL `CREATE TABLE` statement there into something CockroachDB understands.
```

Comment:
Do we have any guidance on how to translate the MySQL CREATE TABLE statement into "something CockroachDB understands? I think this might actually be step 2 with import being step 3?

Reply:
> I'd suggest just using .sql and .txt.
🦊ed, thanks.

> Do we have any guidance on how to translate the MySQL CREATE TABLE statement into "something CockroachDB understands?

Not that I could find, a [search for MySQL](https://www.cockroachlabs.com/docs/search.html?q=mysql) only turned up two results.  However it was reasonably straightforward even for a SQL novice like me (at least in this case).

> I think this might actually be step 2 with import being step 3?

I went with making it part of the "import" step because crafting the `IMPORT` statement basically is the whole step since that one statement both creates the table and loads the data file.  However maybe there's a better way to do it.